flocker/acceptance/testtools.py

```diff
@@ -304,11 +304,15 @@ class Cluster(PRecord):
     :ivar Node control_node: The node running the ``flocker-control``
         service.
     :ivar list nodes: The ``Node`` s in this cluster.
-    :ivar treq: A ``treq`` client.
+
+    :ivar treq: A ``treq`` client, eventually to be completely replaced by
+        ``FlockerClient`` usage.
```

Comment:
Because the client doesn't support the container API yet, so you can't replace all the calls.